README.md

```diff
@@ -0,0 +1,51 @@
+# SES5
+
+> Secure EcmaScript 5
+
+*:warning: This is not an official, working packaging of Google Caja’s SES.
+This is a project attempting to bring SES to Node.js.  There remain a few
+surmountable obstacles before we achieve proper containment.*
+
+SES is a tool that allows mutually suspicious programs to share a single
+EcmaScript 5 compliant JavaScript context without interfering with each
+other.  It does this by freezing everything that is accessible in global
+scope, removing interfaces that would allow programs to interfe with
+each-other, and providing the ability to evaluate arbitrary code in
+isolation.
+
+SES is a part of the Google Caja project.  For JavaScript contexts that
+do not support EcmaScript 5, Caja depends on compiling JavaScript to a
+JavaScript subset with static verification and run-time assertions to
+maintain isolation.  With EcmaScript 5, it is possible to run isolated
+code without a compilation step or run-time checks.
+
+Usage:
+
+```js
+var ses = require('ses5');
```

Comment:
Addressing modules inside and outside a package is different with Node.js. Outside the package, a module like `ses.js` can be addressed as `require("ses5/ses")`, under the name of the package as written in `package.json`. Inside the package, it must be addressed with a relative path, like `require("./ses")`. The `package.json` has a `"main": "./ses.js"` property that indicates that `ses.js` should be loaded when a dependent package refers to a module by the name of the package, i.e., `require("ses")`.